nighres/utils.py

```diff
@@ -12,6 +12,8 @@ def _output_dir_4saving(output_dir=None, rootfile=None):
             # if rootfile is specified, use it's directory
             output_dir = os.path.dirname(rootfile)
 
+    if output_dir=='' : output_dir = './'
```

Comment:
why is this necessary? isn't this what line 10 does (but in an cross-platform way)? Why would output_dir ever be set to an empty string (instead of to None as is default in all functions)?

Reply:
it didn't work, as you need some directory to be set in order to add to it for creating filenames (I think the empty string and None have the same problem, may be in fact the same thing)

Reply:
Still not sure what you mean. If no output_dir, nor a rootfile is given, the expected behaviour is to use the current working directory as output_dir (see line 10).  What problem did you run into exactly?

Reply:
my bad method to create file paths with dir+name failed, but I think I have the solution now using os.path